scripts/apps/relations/services/RelationsService.ts

```diff
@@ -21,7 +21,14 @@ export function RelationsService(api, $q) {
 
         return $q.all(relatedItemsKeys.map((key) => {
             if (isLink(associations[key])) {
-                return api.find('archive', associations[key]._id);
+                return api.find('archive', associations[key]._id)
+                    .catch((response) => {
+                        if (response?.status === 404) {
+                            return null;
```

Comment:
what's the difference between returning `null` and having it rejected? Does the promise become resolved if you return from `catch`?

Reply:
yes it's resolve